azure-local/upgrade/install-solution-upgrade-azure-resource-manager-template.md

```diff
@@ -0,0 +1,264 @@
+> - Installing solution upgrade using ARM template is targeted for at-scale upgrades. This method is intended for IT administrators who have experience managing Azure Local instances. We recommend that you upgrade a system via the Azure portal first, and then use ARM template for subsequent upgrades. To install the solution upgrade via the Azure portal, see [Install solution upgrade on Azure Local](./install-solution-upgrade.md).
+
+## Prerequisites
+
+Before you install the solution upgrade, make sure that you:
+
+- Validate the system using the Environment Checker as per the instructions in [Assess solution upgrade readiness](./validate-solution-upgrade-readiness.md#run-the-validation).
+- Have failover cluster name between 3 to 15 characters.
+- Create an Active Directory Lifecycle Manager (LCM) user account that's a member of the local Administrator group. For instructions, see [Prepare Active Directory for Azure Local deployment](../deploy/deployment-prep-active-directory.md).
+- Have IPv4 network range that matches your host IP address subnet with six, contiguous IP addresses available for new Azure Arc services. Work with your network administrator to ensure that the IP addresses aren't in use and meet the outbound connectivity requirement.
+- Have Azure subscription permissions for [Azure Stack HCI Administrator and Reader](../manage/assign-vm-rbac-roles.md#about-built-in-rbac-roles).  
+
+    :::image type="content" source="media/install-solution-upgrade-azure-resource-manager-template/verify-subscription-permissions-roles.png" alt-text="Screenshot of subscription with permissions assigned to required roles for upgrade." lightbox="./media/install-solution-upgrade-azure-resource-manager-template/verify-subscription-permissions-roles.png":::
+
+::: moniker range="<=azloc-24113"
+
+- For Azure Local 2411.3 and earlier versions, make sure to select the **upgrade-cluster-2411.3** template for upgrade.
+
+::: moniker-end
+
+::: moniker range=">=azloc-2503"
+
+- For Azure Local 2503 and later versions, make sure to select the **upgrade-cluster** template for upgrade.
+
+::: moniker-end
+
+## Before you begin
+
+There are a few things to consider before you begin the solution upgrade process:
+
+- Microsoft supports upgrade applied from Azure Local resource page or by using an ARM template. Use of third-party tools to install upgrades isn't supported.
+- We recommend you perform the solution upgrade during a maintenance window. After the upgrade, host machine might reboot and the workloads will be live migrated, causing brief disconnections.
+<!--Confirm if this point is valid for ARM upgrade- If you have Azure Kubernetes Service (AKS) workloads on Azure Local, wait for the solution upgrade banner to appear on the Azure Local resource page. Then, remove AKS and all AKS hybrid settings before you apply the solution upgrade.-->
```

Comment:
For now publish with the above commented out. Remove it in followup PR once we have a response from Niranjan.